docs/adfs/README.md

```diff
@@ -111,6 +111,16 @@ module.exports = passport;
 ```
 # Troubleshooting
 
+## Errors you might encounter
+
+|Error   | Explanation  | Mitigation  |
+|--------|---|---|
+| `SAML assertion not yet valid`  | This means that there is a clock skew between your server time and your saml provider time. There might be a better way by fiddling with the ClockSkew You can disable clock skew validation with a value of `-1`.  | `{ acceptedClockSkewMs: -1 }`  |
```

Comment:
I think it's helpful to have a chart of possible problems, with explanations and mitigations, so thanks for starting that. Some comments:

 * SAML should be also be written as "SAML", not "saml".
 * Checking the clock skew is an important security feature. Without the check, there is increased vulnerability to reply attacks. So simply advising users to disable the security check is poor advise. First, you should try to fix the clocks on your server and the providers to match UTC closely if possible. If that's not possible, accepting the minimum clock skew between the servers is the nex best option. Disabling the clock skew should be a last resort.